source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -89,3 +89,10 @@ Removing a Library Dependency
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 To remove a library dependency from a project, select **Manage Current Libraries** from the **Manage Vendor Libraries** menu, check the box for any libraries to uninstall and click OK. These libraries will be removed as dependencies from the project.
+
+Command-Line Interface (Online)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Adding a vendor library dependency from the vendor URL can also be done through the command-line via a gradle task. Open a command-line instance at the project root, and enter `gradlew vendordep --url=<url>` where `<url>` is the vendor JSON URL. This will add the vendor library dependency JSON file to the **vendordeps** folder of the project. Vendor libraries can be updated the same way.
+The `vendordep` gradle task can also fetch vendordep JSONs from the user `wpilib` folder. To do so, pass `$FRCLOCAL/Filename.json` as the file URL. For example, `gradlew vendordep --url=$FRCLOCAL/WPILibNewCommands.json` will fetch the JSON for the new command-based framework from the user `wpilib` folder.
```

Comment:
```suggestion
The `vendordep` gradle task can also fetch vendordep JSONs from the user `wpilib` folder. To do so, pass `$FRCLOCAL/Filename.json` as the file URL. For example, `gradlew vendordep --url=$FRCLOCAL/WPILibNewCommands.json` will fetch the JSON for the new command-based framework.
```
@Daltz333 do you think that this last phrase is redundant?

Reply:
Yes.